helm/dendrite/templates/ingress.yaml

```diff
@@ -1,8 +1,8 @@
 {{- if .Values.ingress.enabled -}}
 {{- $fullName := include "dendrite.fullname" . -}}
 {{- $serverNameHost := .Values.dendrite_config.global.server_name -}}
-{{- $wellKnownServerHost := default $serverNameHost (regexFind "^[^:]+" .Values.dendrite_config.global.well_known_server_name) -}}
-{{- $wellKnownClientHost := default $serverNameHost (regexFind "^[^:]+" .Values.dendrite_config.global.well_known_client_name) -}}
+{{- $wellKnownServerHost := default $serverNameHost (regexFind "^(\\[.+\\])?[^:]*" .Values.dendrite_config.global.well_known_server_name) -}}
```

Comment:
This seems to work for actual domains an IPv4, but not IPv6?
https://regex101.com/r/FEMTE1/1

Reply:
If you're testing in an online platform like that, you're going to have to remove the additional escaping character.

Reply:
I feel stupid.. You're absolutely right!